staking_deposit/cli/generate_bls_to_execution_change.py

```diff
@@ -0,0 +1,195 @@
+    ),
+    help=lambda: load_text(['arg_bls_withdrawal_credentials_list', 'help'], func=FUNC_NAME),
+    param_decls='--bls_withdrawal_credentials_list',
+    prompt=lambda: load_text(['arg_bls_withdrawal_credentials_list', 'prompt'], func=FUNC_NAME),
+)
+# TODO: retype confirmation
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda address: validate_eth1_withdrawal_address(None, None, address),
+        lambda: load_text(['arg_execution_address', 'prompt'], func=FUNC_NAME),
+    ),
+    help=lambda: load_text(['arg_execution_address', 'help'], func=FUNC_NAME),
+    param_decls='--execution_address',
+    prompt=lambda: load_text(['arg_execution_address', 'prompt'], func=FUNC_NAME),
+)
+@jit_option(
+    # Only for devnet tests
+    default=None,
+    help="[DEVNET ONLY] Set specific GENESIS_FORK_VERSION value",
+    param_decls='--devnet_chain_setting',
+)
+@click.pass_context
+def generate_bls_to_execution_change(
+        ctx: click.Context,
+        bls_to_execution_changes_folder: str,
+        chain: str,
+        mnemonic: str,
+        mnemonic_password: str,
+        validator_start_index: int,
+        validator_indices: Sequence[int],
+        bls_withdrawal_credentials_list: Sequence[bytes],
+        execution_address: HexAddress,
+        devnet_chain_setting: str,
+        **kwargs: Any) -> None:
+    # Generate folder
+    bls_to_execution_changes_folder = os.path.join(
+        bls_to_execution_changes_folder,
+        DEFAULT_BLS_TO_EXECUTION_CHANGES_FOLDER_NAME,
+    )
+    if not os.path.exists(bls_to_execution_changes_folder):
+        os.mkdir(bls_to_execution_changes_folder)
+
+    # Get chain setting
+    chain_setting = get_chain_setting(chain)
+
+    if devnet_chain_setting is not None:
+        click.echo('\n%s\n' % '**[Warning] Using devnet chain setting to generate the SignedBLSToExecutionChange.**\t')
+        devnet_chain_setting_dict = json.loads(devnet_chain_setting)
+        chain_setting = get_devnet_chain_setting(
+            network_name=devnet_chain_setting_dict['network_name'],
+            genesis_fork_version=devnet_chain_setting_dict['genesis_fork_version'],
+            genesis_validator_root=devnet_chain_setting_dict['genesis_validator_root'],
+        )
+
+    if len(validator_indices) != len(bls_withdrawal_credentials_list):
+        raise ValueError(
+            "The size of `validator_indinces` (%d) should be as same as `bls_withdrawal_credentials_list` (%d)."
+            % (len(validator_indices), len(bls_withdrawal_credentials_list))
+        )
+
+    num_validators = len(validator_indices)
+    amounts = [MAX_DEPOSIT_AMOUNT] * num_validators
+
+    credentials = CredentialList.from_mnemonic(
+        mnemonic=mnemonic,
+        mnemonic_password=mnemonic_password,
+        num_keys=num_validators,
+        amounts=amounts,
+        chain_setting=chain_setting,
+        start_index=validator_start_index,
+        hex_eth1_withdrawal_address=execution_address,
```

Comment:
can both non hex and hex be handled? or if the error for non hex lets the user know,I believe some users will get the withdrawal pub key from the intial deposit.json which doesn't have 0x

Reply:
`bls_withdrawal_credentials_list` can be with `0x` prefix or no `0x`.
`eth1_withdrawal_address` requires the checksum form and `0x` prefix.

Reply:
@hwwhww Does this mean if the user pastes an all lowercase address it will be blocked?

Reply:
@wackerow 
in a very edge case, all-number address is a valid checksum address 😄 but I understand what you mean, yes, the input addresses have to be checksummed and capital letters matter.